src/hooks/usePoolTickData.ts

```diff
@@ -140,8 +142,17 @@ function useTicksFromSubgraph(
   currencyB: Currency | undefined,
   feeAmount: FeeAmount | undefined
 ) {
+  const { chainId } = useWeb3React()
```

Comment:
I think you want `import useActiveWeb3React from 'hooks/useActiveWeb3React'` here

Reply:
I believe we refactored useActiveWeb3React in [this commit](https://github.com/Uniswap/interface/commit/322cdaf88897fce292b29f41fda4f09186f3529f)